packages/html-spec/src/listview/listview.json

```diff
@@ -0,0 +1,158 @@
+{
+    "$schema": "../../schemas/component.schema.json",
+    "title": "Kendo Listview HTML Spec",
+    "component": "listview",
+    "selector": [
+        ".k-listview"
+    ],
+    "states": [
+        {
+            "name": "disabled",
+            "selector": [
+                ".k-disabled"
+            ]
+        }
+    ],
+    "options": [
+        {
+            "name": "layout",
```

Comment:
since layout and flex-direction are utility classes and not component-specific ones, I don't think they should be added to the JSON.

Reply:
the classes are need for other listView option which is flex-direction